internal/frontend/http/templates/wizard-final.html

```diff
@@ -0,0 +1,190 @@
+{{ define "wizard-final" }}
+{{ template "hidden-inputs" . }}
+{{ template "header" "Schematic Ready" }}
```

Comment:
Maybe? But let's see what our native speakers say :)

```suggestion
{{ template "header" "Schematic is Ready" }}
```

Reply:
:man_shrugging: